pkg/cri/config/config.go

```diff
@@ -84,11 +84,6 @@ type ContainerdConfig struct {
 	// Snapshotter is the snapshotter used by containerd.
 	Snapshotter string `toml:"snapshotter" json:"snapshotter"`
 
-	// PullImageRuntimeAnnotationKey specifies the annotation key to get runtime type from PullImageRequest.SandboxConfig.Annotations
-	// Ideally we would want the runtime to be passed in as a field in PullImageRequest, which requires kubelet support.
-	// See https://github.com/containerd/containerd/issues/6657
-	PullImageRuntimeAnnotationKey string `toml:"pull_image_runtime_annotation_key" json:"pullImageRuntimeAnnotationKey"`
```

Comment:
This is a breaking change right? Existing tomls will now fail

Reply:
This is already removed in the latest commit. Also I don't think existing toml will fail if a field is not specified, unless my understanding is off